src/alf/themes.ts

```diff
@@ -132,21 +132,21 @@ export const darkPalette: Palette = {
 
 export const dimPalette: Palette = {
   ...darkPalette,
-  black: tokens.color.gray_1000,
+  black: `hsl(211, 28%, 12%)`,
 
-  contrast_25: tokens.color.gray_975,
-  contrast_50: tokens.color.gray_950,
-  contrast_100: tokens.color.gray_900,
-  contrast_200: tokens.color.gray_800,
-  contrast_300: tokens.color.gray_700,
-  contrast_400: tokens.color.gray_600,
-  contrast_500: tokens.color.gray_500,
-  contrast_600: tokens.color.gray_400,
-  contrast_700: tokens.color.gray_300,
-  contrast_800: tokens.color.gray_200,
-  contrast_900: tokens.color.gray_100,
-  contrast_950: tokens.color.gray_50,
-  contrast_975: tokens.color.gray_25,
+  contrast_25: `hsl(211, 28%, 14%)`,
+  contrast_50: `hsl(211, 28%, 17%)`,
+  contrast_100: `hsl(211, 28%, 23%)`,
```

Comment:
We use this for the border in `lib/themes.ts`. This makes sense I think whenever we are dealing with solid white/black, but I feel like in this case we might want something a bit less "blue"? Maybe it's just me, but on some monitors this is just slightly too blended in with the background for the borders.